cdap-data-fabric/src/main/java/io/cdap/cdap/store/StoreDefinition.java

```diff
@@ -392,8 +395,12 @@ public static final class AppMetadataStore {
         .withFields(Fields.stringType(NAMESPACE_FIELD),
                     Fields.stringType(APPLICATION_FIELD),
                     Fields.stringType(VERSION_FIELD),
-                    Fields.stringType(APPLICATION_DATA_FIELD))
+                    Fields.stringType(APPLICATION_DATA_FIELD),
+                    Fields.longType(CREATION_TIME_FIELD),
+                    Fields.stringType(OWNER_FIELD),
+                    Fields.stringType(LATEST_FIELD))
         .withPrimaryKeys(NAMESPACE_FIELD, APPLICATION_FIELD, VERSION_FIELD)
+        .withIndexes(LATEST_FIELD)
```

Comment:
I thought we also agreed upon that we're adding `created` as the indexed field for app listing purpose?